src/strategies/staked-defi-balance/index.ts

```diff
@@ -15,40 +19,80 @@ export async function strategy(
   options,
   snapshot
 ): Promise<Record<string, number>> {
-  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
-  const stakingPoolContractAddress = options.stakingPoolContractAddress;
-  const abi = options.methodABI;
-
-  const stakingCalls = addresses.map((address) => {
-    return [
-      stakingPoolContractAddress,
-      'stakeOf',
-      [options.tokenContractAddress, address]
-    ];
-  });
+  const addressScores = {};
 
-  const stakes = await multicall(network, provider, abi, stakingCalls, {
-    blockTag
-  });
+  for (const params of options) {
```

Comment:
please add a limit to this :) there is a limit of 5 multicalls, refer https://github.com/snapshot-labs/snapshot-strategies#code  if we pass more, it can trigger more multicalls.

Reply:
@ChaituVR doing it now

Reply:
@ChaituVR added check in line 22 through 25